app/services/enqueue_index_jobs.rb

```diff
@@ -0,0 +1,87 @@
+# Indexes a book version thru the SQS queuing system
+class EnqueueIndexJobs
+  def initialize
+    @todo_jobs_queue = TodoJobsQueue.new
+    @new_create_index_jobs = 0
+    @new_delete_index_jobs = 0
+    @worker_asg = AutoScalingGroup.new(worker_asg_name)
+  end
+
+  def call
+    released_book_ids.each do |book_id|
+      ACTIVE_INDEXING_STRATEGIES.each do |indexing_version|
```

Comment:
i used indexing_version because that what it was.   a strategy is the indexing object/class.  agree we should be consistent.  are u ok with calling this ACTIVE_INDEXING_VERSIONS?